### ISSUE TITLE
Global brightness setting

### ISSUE THREAD

**Issue by tipeon** [nvdweem/PCPanel#18]
My brand new PC Panel tends to be too bright, so I've had to change the color setting to 5% brightness. (yes, I live in a dimly lit cave)

It works pretty well, but
1. It is tedious to change every single color by hand
2. In the GUI, 5% is too dark, all colors are rendered as gray

I think that a good solution to this problem would be a global brightness setting that applies only to the LEDs in the PC Panel (not the GUI)

**Comment by nvdweem** [comment]
Good idea. Having a global brightness slider would allow removing the option from the 'Full body > Rainbow/Wave/Breath' options and allows it to be applied to any selected color in any of the other options. 
It would also open up the option to have a slider/knob control the brightness as well.

**Comment by nvdweem** [comment]
I kept the brightness setting for the animations and added a global slider on the main page. The slider can also be controlled by a dial/slider. I did notice that there is a verly low range where the brightness percentage makes much sense (30-100% look like full brightness).

Let me know if you have any additional suggestions.

**Comment by tipeon** [comment]
Hey, it's nice that you implemented my suggestion so fast. 👍 

I had a look at the latest snapshot, and it works pretty well.

However, there's two things I noticed
- some actions seem to reset global brightness to 100% (I noticed this happens when I switch tabs in the lighting dialog, or when I check/uncheck Mute override)
- speaking of mute override, there seems to be a regression. At first, it stayed in the "override" state no matter what I did, now it won't activate at all ... I'll try to find a reproducible way to trigger the problem when I have more time.

**Comment by tipeon** [comment]
OK, that last one was just me being dumb. Mute override works fine, I was just fiddling with configuration in the wrong tab.

I found another weird thing: When trying to assign the global brightness to a knob, I turned on logarithmic scaling.
That made PCPanel freak out and fail to apply the setting. After a quick stop/restart of the software, everything went back to normal. (so at least, the setting is saved and reapplied on startup)
